bundles/org.eclipse.equinox.launcher.tests/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,10 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Tests
+Bundle-SymbolicName: org.eclipse.equinox.launcher.tests
+Bundle-Version: 1.0.0.qualifier
+Fragment-Host: org.eclipse.equinox.launcher
```

Comment:
just wondering why we are making it fragment..

Reply:
This allows us to access package private classes in the `org.eclipse.equinox.launcher` bundle in the this test fragment and therefore avoids the symbolic link.
Fragment and host share the same classloader, therefore if you put classes in a fragment they can also access package-private elements in the same package of the host.